java/gui/src/main/java/com/squareup/subzero/SubzeroGui.java

```diff
@@ -54,6 +55,9 @@ public class SubzeroGui {
   // Almost always you want to talk to subzero on localhost
   @Parameter(names = "--hostname") public String hostname = "localhost";
 
+  // Wallet file location
+  @Parameter(names = "--wallet-file") public String walletFilePath = "/data/app/subzero/wallets/";
```

Comment:
nit: I think this should be called `--wallet-dir` or similar since it's actually the path to the wallet directory

Reply:
nit: and similarly call the variable `walletDirectory` or something like that, since it's neither a Path object (it's a String), nor names the wallet file.